packages/stark-ui/assets/styles/_header.scss

```diff
@@ -65,21 +65,6 @@
         }
       }
     }
-
-    .stark-app-bar-page-title {
```

Comment:
We cannot remove this rules in `stark-ui`. This is needed for any app (maybe not in our showcase) to display the title of the application which is not necessary the same as the logo.